src/app/shared/components/map/map.component.ts

```diff
@@ -11,38 +11,62 @@ import {
 } from "@angular/core";
 import * as L from "leaflet";
 import "leaflet.fullscreen";
-import { Subscription } from "rxjs";
 import { MapCoordinates, MarkerInfo } from "./map-coordinates.model";
 import { OpenStreetMapProvider, GeoSearchControl } from "leaflet-geosearch";
 import { TranslateService } from "@ngx-translate/core";
 import moment from "moment";
 import "leaflet.markercluster";
+import "proj4leaflet";
+import { environment } from "@environments/environment";
 
 @Component({
     selector: "app-map",
     templateUrl: "./map.component.html",
     styleUrls: ["./map.component.scss"],
 })
 export class MapComponent implements OnInit, AfterViewInit, OnChanges, OnDestroy {
-    private map;
+    private satelliteCenterLat = 56.07900207957074;
```

Comment:
Probably add a commen on what this is, OR move it to a named constant in a constants file